extract_project_descriptions_and_public_interests.py

```diff
@@ -0,0 +1,75 @@
+""" This file is for getting the project descriptions and 
+public interests from xml and save them to a csv file. """
+import time
+import re
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def extract_project_description_and_public_interest(url_df):
+    """Function to extract project description and public interest from the xml."""
+
+    #initialize lists
+    list_of_project_ids = []
+    list_of_project_descriptions = []
+    list_of_public_interest = []
+
+    #loop through the urls in the csv
+    for url in url_df:
+        #get project_id from url and save to the list
+        project_id = url.split("id=",1)[1]
+        list_of_project_ids.append(project_id)
+
+        #get the url content
+        url = requests.get(url, timeout=5)
+        soup = BeautifulSoup(url.content, 'html.parser')
+
+        #extract the project description from xml and save to the list
+        project_description = soup.find("div", class_="project-description").text
+        #remove the titles "Hlasovací video" and "Realizační video" from the description
+        project_description = re.sub('Hlasovací video', '', project_description)
+        project_description = re.sub('Realizační video', '', project_description)
```

Comment:
make it oneliner
project_description = re.sub('(Hlasovací|Realizační) video', '', project_description)